src/swaps/SwapProvider.ts

```diff
@@ -40,6 +40,11 @@ export abstract class SwapProvider {
    */
   public abstract getSupportedPairs({ network }: { network: Network }): Promise<PairData[]>;
 
+    /**
+   * Get min swap amount for given swap provider
+   */
+  public abstract getMin(quoteRequest: QuoteRequest) : number | any
```

Comment:
`| any` 
Why? The possible values for getting the minimum should be number or in the case of no minimum, null or undefined

Reply:
I think that liquality provider is returning some kind of object in getMin, but I will check if I can refactor this piece of code to make it consistent

Reply:
Resolved getMin is returning BigNumber each time, any removed